api/src/indigo_loaders.cpp

```diff
@@ -52,6 +89,35 @@ IndigoSdfLoader::~IndigoSdfLoader()
 {
 }
 
+IndigoObject* IndigoSdfLoader::next()
+{
+    if (sdf_loader->isEOF())
+        return 0;
+    
+    int counter = sdf_loader->currentNumber();
+    long long offset = sdf_loader->tell();
+    
+    sdf_loader->readNext();
+    
+    return new IndigoRdfMolecule(sdf_loader->data, sdf_loader->properties, counter, offset);
+}
+
+IndigoObject* IndigoSdfLoader::at(int index)
+{
+    sdf_loader->readAt(index);
+    return new IndigoRdfMolecule(sdf_loader->data, sdf_loader->properties, index, 0LL);
+}
+
+bool IndigoSdfLoader::hasNext()
+{
+    return !sdf_loader->isEOF();
+}
+
+long long IndigoSdfLoader::tell()
+{
+    return sdf_loader->tell();
+}
+

```

Comment:
It's hard to identify among a large pull-request that these strings are just moved to better position.
Maybe keep them in-place to improve traceability?

Reply:
fixed